hadoop-ozone/dist/src/main/compose/testlib.sh

```diff
@@ -398,7 +398,7 @@ run_rebot() {
 
   shift 2
 
-  local tempdir="$(mktemp -d --suffix rebot -p "${output_dir}")"
+  local tempdir="$(mktemp -d "${output_dir}"/rebot-XXXXXX)"
```

Comment:
What is the `rebot-XXXXXX` part? Is `XXXXX` a placeholder or is that the actual filename?